pandas/tests/arrays/integer/test_arithmetic.py

```diff
@@ -261,3 +261,41 @@ def test_reduce_to_float(op):
         index=pd.Index(["a", "b"], name="A"),
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("dtype", ["Int64", "Int32", "Int16", "Int8"])
+@pytest.mark.parametrize(
+    "source, target",
+    [
+        ([1, 2, 3], [-1, -2, -3]),
+        ([1, 2, None], [-1, -2, None]),
+        ([-1, 0, 1], [1, 0, -1]),
+    ],
+)
+def test_unary_minus_nullable_int(dtype, source, target):
+    arr = pd.array(source, dtype=dtype)
+    result = -arr
+    expected = pd.array(target, dtype=dtype)
+    tm.assert_extension_array_equal(result, expected)
+
+
+@pytest.mark.parametrize("dtype", ["Int64", "Int32", "Int16", "Int8"])
```

Comment:
can you use the any_nullable_int_dtype fixture instead

Reply:
We can't because these are signed integers, could create another fixture for signed if that makes sense